### ISSUE TITLE
Allow setting deadlines for the students and the evaluators. Better accountable system.

### ISSUE THREAD

**Issue by puneethrj** [fibasile/nueval-project#51]
Allow evaluators to set deadline for the students to finish working on the feedback. It can be for a specific feedback, specific assignment or for the entire review.

Evaluators should be made more accountable. Allow a check box feature where the student can notify the evaluator that the s/he has worked on the suggestions. The evaluator will from then on be given a minimum time until which s/he should respond and check on the work done. 

This system could be made better if the evaluator and student could put down their work time/commitment(example : 2PM to 4PM everyday in their respective time zone) and the process could be automated. The output will be that the evaluator and the student get a notification helping them to better stick to deadlines. The deadline should also be meaningful. It means that if a student is left with a feedback to be worked on the 10th hour and he finishes working on it on the 11th hour, by the time the absolute deadline(19th 23:59 for this year) arrives the evaluator should be able to review it and mark the progress of the student. Or a different final deadline could be worked depending on a case, lets say the evaluator gets 6 hours from the time the student has made the update to review it.

If the evaluator misses a deadline for looking on a feedback/assignment/review it  should automatically pass on to secondary evaluation team/person who's only job is to check if the feedback suggested has been worked on by the student and accordingly mark the progress of the student and not to provide any extra comment on any other aspect of the assignment, that means s/he is only responsible to see if that particular issue and doesn't have to review more. People who want less load or possibly some local instructors could opt to volunteer for this kind of a responsibility.

in short
1. Option to set deadlines by the evaluators.
2. System that makes evaluators accountable for giving feedback and review.
3. Engage volunteers to better handle the load.
4. Checkbox to mark acknowledgement and resolve of a feedback (https://github.com/fibasile/nueval-project/issues/34)
